spec.html

```diff
@@ -35715,7 +35715,7 @@ <h1>Static Semantics: Early Errors</h1>
             It is a Syntax Error if CountLeftCapturingParensWithin(|Pattern|) ≥ 2<sup>32</sup> - 1.
           </li>
           <li>
-            It is a Syntax Error if |Pattern| contains two or more |GroupSpecifier|s for which CapturingGroupName of |GroupSpecifier| is the same.
+            It is a Syntax Error if |Pattern| contains two distinct |GroupSpecifier|s _x_ and _y_ for which CapturingGroupName(_x_) is the same as CapturingGroupName(_y_) and such that CanBothParticipate(_x_, _y_) is *true*.
```

Comment:
1) PR #2877 says to avoid "is the same as", use "is" for comparing Strings.

2) It's rare (though not unheard of) to invoke an SDO with parenthesis notation. On the other hand, using the normal notation would put `CapturingGroupName of _y_` on the RHS of `is`, which I don't think we *ever* do.

3) `... for which A and such that B` is odd. Seems like either the two arms should agree re "for which" vs "such that", or else just use one that covers both arms: `... such that A and B`

```suggestion
            It is a Syntax Error if |Pattern| contains two distinct |GroupSpecifier|s _x_ and _y_ such that CapturingGroupName(_x_) is CapturingGroupName(_y_) and CanBothParticipate(_x_, _y_) is *true*.
```

Reply:
Changed to

> It is a Syntax Error if |Pattern| contains two distinct |GroupSpecifier|s `_x_` and `_y_` such that the CapturingGroupName of `_x_` is the CapturingGroupName of `_y_` and such that CanBothParticipate(`_x_`, `_y_`) is `*true*`.